packages/contact-center/task/src/CallControl/call-control.presentational.tsx

```diff
@@ -1,9 +1,10 @@
 import React, {useEffect, useState} from 'react';
-import {WrapupCodes} from '@webex/cc-store';
 
 import {CallControlPresentationalProps} from '../task.types';
 import './call-control.styles.scss';
-import {ButtonPill} from '@momentum-ui/react-collaboration';
+import {Button, Icon} from '@momentum-design/components/dist/react';
```

Comment:
Why are we using the @momentum-design components for these? Shouldn't everything come from react-collaboration?

Reply:
We have [<ButtonCircle />](https://momentum-design.github.io/momentum-react-v2/?path=/story/momentum-ui-buttoncircle--example) as well as [<Icon/>](https://momentum-design.github.io/momentum-react-v2/?path=/story/momentum-ui-icon--example)